Add Cloud Docs (#686)
graknlabs/docs: cloud-src/modules/ROOT/pages/deployments/management.adoc

= Deployment management
:keywords: typedb, deployment, cluster, cloud, management, create, destroy, suspend, resume, lifecycle, life cycle
:longTailKeywords: create deployment in typedb cloud, destroy deployment, stop deployment, resume deployment
:pageTitle: Deployment management
:summary: How to manage deployments in TypeDB Cloud.
:experimental:

A TypeDB Cloud deployment is a TypeDB Enterprise cluster that TypeDB Cloud manages.

> image::cloud::cloud.png[TypeDB Cloud web interface,width = 90%, link=self]

Author: We should probably open images only in new tab

4. Click the btn:[Suspend] button at the top right toolbar.
5. Confirm suspension by clicking the btn:[Confirm] button in the confirmation window.
====

== Resume a deployment

To resume a suspended deployment:

1. xref:cloud::user-management/organization.adoc#_select[Select] the Organization with the deployment.
2. Open the Deployments page from the navigation menu on the left.
3. Select the deployment or deployments to resume by ticking the checkbox in the respective lines on the left.
4. Click the btn:[Resume] button at the top toolbar.
5. Confirm resuming by clicking the btn:[Confirm] button in the confirmation window.

[caption=""]
.Alternatively
====
1. xref:cloud::user-management/organization.adoc#_select[Select] the Organization with the deployment.
2. Open the Deployments page from the navigation menu on the left.
3. Select the deployment to resume by clicking on it.
4. Click the btn:[Resume] button at the top right toolbar.
5. Confirm resuming by clicking the btn:[Confirm] button in the confirmation window.
====

== Destroy a deployment

To destroy a Deployment means to suspend it and then delete all its data.

[WARNING]
====

> This action is irreversible and may lead to a loss of information stored in the deployment.

Reviewer: ```suggestion
This action is irreversible and will destroy all information stored in the deployment.
```

Author: Do we really destroy the information/data? Like guaranteed erasing? I would be careful here for legal reasons and just use "all information stored in the deployment will be lost".

Author: Adjusted in https://github.com/vaticle/typedb-docs/pull/686/commits/9dd696e44c70151070a284dc0e90b7708c493fb7
